### DISCUSSION TITLE
is there a way to install Simpleton ui on windows 10 without the WSL or Virtualbox method?

### DISCUSSION THREAD

**Discussion by zickcunt** [apertium/organisation#41] [General]
i have tried to follow the wiki but when it says "After unzipping, run the application in the bin folder called apertium-simpleton." i dont find the bin folder only a .pro file?

**Comment by TinoDidriksen** [comment]
That means you downloaded the source code, not the binary. Unfortunately the Windows binaries doesn't work right now. I put some effort into the [Windows builds this week](https://github.com/apertium/packaging/commit/3514501766d91a4ae8c8b789fa87c267be8c3941), but while the builds all now succeed, they don't all run, and I haven't had time to debug that further yet.

(see also https://github.com/orgs/apertium/discussions/39)
